assets/rnadeseq_report.Rmd

```diff
@@ -2858,50 +3132,61 @@ quant_cite,
 ))
 } else if (params$input_type == "smrnaseq"){
 cat(paste0(
-"## smRNA-seq data analysis
+"## Primary Bioinformatics
+
+### smRNA-seq data analysis {#smrnaseq-methods}
 
-The Nextflow-based nf-core pipeline `smrnaseq ", as.character(version["Workflow"][[1]]["nf-core/smrnaseq"]),
-"` [^6] was used for the smRNA-seq Bioinfomatics analysis. `FASTQC ",
-as.character(version["FASTQC"][[1]]["fastqc"]),
-"` [^3] [@andrews2010fastqc] was used to determine the quality of the miRNA samples. ",
-ifelse("TRIMGALORE" %in% names(version), paste0("Subsequently, adapter trimming was conducted with `Trim Galore ",
+The Nextflow-based [nf-core/smrnaseq](https://nf-co.re/smrnaseq/) pipeline, revision ", as.character(version["Workflow"][[1]]["nf-core/smrnaseq"]),
```

Comment:
I am sure there are some moe `revision` around, I think all need to be unified and rather version or v or such.

Reply:
Will check on that